scipy/linalg/_basic.py

```diff
@@ -164,6 +168,10 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
 
     # Backwards compatibility - old keyword.
     if sym_pos:
+        message = ("This key is deprecated and ``assume_a = 'pos'`` keyword is"
```

Comment:
Usually in warnings and errors we use single backticks for code (perhaps because the double-backtick requirement of reStructuredText is less common?). E.g.
```
np.random.rand(10.4)  # 'float' object cannot be interpreted as an integer
```